src/ripple/app/misc/TxQ.h

```diff
@@ -319,6 +319,19 @@ class TxQ
     Metrics
     getMetrics(OpenView const& view) const;
 
+    /**
+     * @brief Returns minimum required fee for tx and two sequences:
+     *        first vaild sequence for this account in current ledger
+     *        and first available sequence for transaction
+     * @param view current open ledger
+     * @param tx the transaction
+     * @return minimum required fee, first sequence in the ledger
+     *        and first available sequence
+     */
+    std::tuple<XRPAmount, std::uint32_t, std::uint32_t>
```

Comment:
I'd return a struct here, not a tuple. This tuple has type fields of the same type, both for sequences. I'd rather use a struct to help keep these fields straight.

Reply:
Done